### PR TITLE
feat(choice tile): new component specs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`<hx-choicetile>`:  implementing color palette 2.0 styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF2153

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by BY00565233 — APPROVED** [review]
DEV LGTM

**Review by danielmdesigns — CHANGES_REQUESTED** [review]
I realized the design spec was not correct for Option Tiles. I have updated Abstract Style Guide specs.

1. Tile icons should match that of the state (hover, pressed)
2. I also included a disabled state

My apologies for overlooking these. Please update these two and let me know when I can take another look. That should solve any issues I have.

**Comment by lalithkarikelli** [comment]
> I realized the design spec was not correct for Option Tiles. I have updated Abstract Style Guide specs.
> 
> 1. Tile icons should match that of the state (hover, pressed)
> 2. I also included a disabled state
> 
> My apologies for overlooking these. Please update these two and let me know when I can take another look. That should solve any issues I have.

@danielmdesigns i have pushed the changes , please review at https://deploy-preview-814--helix-ui.netlify.app/components/choice-tile . Thanks !

**Review by 100stacks — CHANGES_REQUESTED** [review]
@lalithkarikelli, it looks like we have legacy box shadow focus styles on Choice Tile:

Screen Shot 2021-01-13 at 10 16 29 AM

Screen Shot 2021-01-13 at 10 16 40 AM

#### Please see https://deploy-preview-814--helix-ui.netlify.app/components/choice-tile/test.html

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss [hunk 1]
```diff
@@ -8,8 +8,20 @@
     box-shadow: $focus-glow;
```
Comment (100stacks) [inline src/scss/components/choice-tile/mixins/_hxChoiceTile.scss:8]: Please update to:

```css
box-shadow: 0 0 4px rgba($magenta-700, 0.5);
```

**Review by 100stacks — CHANGES_REQUESTED** [review]
Please update the focus state `box-shadow`.

File: src/scss/components/choice-tile/mixins/_hxChoiceTile.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/choice-tile/mixins/_hxChoiceTile.scss]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM

File: docs/components/icon/index.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h2 id="status-icons">Status Icons</h2>
 
     <div class="example">
       <div>
-        <hx-icon type="info-circle" style="color: #296CDC" class="size-2x"></hx-icon>
+        <hx-icon type="information" style="color: #296CDC" class="size-2x"></hx-icon>
```
Comment (100stacks) [inline docs/components/icon/index.html:67]: From PR #823 - update the demo to use the new icon type.

File: docs/components/icon/index.html [hunk 3]
```diff
@@ -74,7 +74,7 @@ <h2 id="status-icons">Status Icons</h2>
 
       <footer>
         {% code 'html' %}
-          <hx-icon type="info-circle" style="color: #296CDC"></hx-icon>
+          <hx-icon type="information" style="color: #296CDC"></hx-icon>
```
Comment (100stacks) [inline docs/components/icon/index.html:77]: From PR #823 - update the demo to use the new icon type.

**Review by danielmdesigns — CHANGES_REQUESTED** [review]
We need to update the invalid color to match status color #D6251F

**Review by 100stacks — COMMENTED** [review]
> We need to update the invalid color to match status color #D6251F

@danielmdesigns, invalid status color updated.  Please see update.

**Review by 100stacks — APPROVED** [review]
@danielmdesigns, I'm going to go ahead and merge this PR #814 so that we can include it in today's release.  

We can make any other changes in a fast follow.  Thanks.

**Pull request merged**